src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -73,12 +78,17 @@ def load_arguments(self, _):
         c.argument('key_permissions', arg_type=get_enum_type(KeyPermissions), metavar='PERM', nargs='*', help='Space-separated list of key permissions to assign.', validator=validate_policy_permissions)
         c.argument('secret_permissions', arg_type=get_enum_type(SecretPermissions), metavar='PERM', nargs='*', help='Space-separated list of secret permissions to assign.')
         c.argument('certificate_permissions', arg_type=get_enum_type(CertificatePermissions), metavar='PERM', nargs='*', help='Space-separated list of certificate permissions to assign.')
+        c.argument('storage_permissions', arg_type=get_enum_type(StoragePermissions), metavar='PERM', nargs='*', help='Space-separated list of storage permissions to assign.')
```

Comment:
This follows the same behavior as the rest of the permissions args.  It is valid to specify no permissions as this will remove any already existing permissions.